ext/opentelemetry-ext-psycopg2/README.rst

```diff
@@ -0,0 +1,25 @@
+OpenTelemetry Psycopg integration
+=================================
+
+The integration with PostgreSQL supports the `Psycopg`_ library and is specified
+to ``trace_integration`` using ``'PostgreSQL'``.
+
+.. Psycopg: http://initd.org/psycopg/
+
+Usage
+-----
+
+.. code:: python
+    import psycopg2
+    from opentelemetry.trace import tracer
+    from opentelemetry.trace.ext.postgresql import trace_integration
+    trace_integration(tracer())
```

Comment:
I believe we changed the way we construct `tracer` with [this PR](https://github.com/open-telemetry/opentelemetry-python/pull/301). Might also want to change the other integration examples.